.github/workflows/auto-merge-dependabot.yaml

```diff
@@ -0,0 +1,17 @@
+name: Auto merge dependabot
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  auto-merge-dependabot:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        ## https://github.com/ahmadnassri/action-dependabot-auto-merge
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
```

Comment:
instead of using an intransparent third-party action, check https://github.com/SciCatProject/frontend/pull/1216/files#diff-2fe48d1021006468ac63263583b5db615dd5e63a39ebff862cc4e2f06b59caa1 

its just  3 or 4 lines more without, and we have much more control.

Reply:
I agree with @bpedersen2. Con we try the other solution?

Reply:
With that solution it seems like we need to enable auto-merge, I'm not sure if that's a good idea.
I need to test the script further

Reply:
My version just sets  automerge per pull-request if it is coming from dependabot.  So changes get merge if tests pass.

Reply:
I see, enable auto merge will check if all the requirements passed. Then we just need to include an PR approved flow it should work, because with current merge rule at least 1 approval is required